test/unit/compute_resources/cr_test_helpers.rb

```diff
@@ -0,0 +1,27 @@
+module CRTestHelpers
```

Comment:
Small quibble, `ComputeResourceTestHelper` seems more idiomatic & friendly for newcomers who might not know what a `CR` is.